.pylintrc

```diff
@@ -0,0 +1,399 @@
+[MASTER]
```

Comment:
Is this a standard .pylintrc file?

Reply:
This is the .pylintrc from tensorflow/k8s. I believe that was a standard .pylintrc that was then lightly modified for our specific use case.